README.md

```diff
@@ -35,6 +37,27 @@ To get a local copy up and running follow these simple example steps.
 
 ## 🤝 Contributing
 
```

Comment:
- [ ] [OPTIONAL] Your readme file looks descriptive and professional. I would like to recommend you update it with explicit steps on how to clone and run your tests locally, as this will be of help to non-technical users. :smile:

Reply:
Thank you for this user-friendly recommendation 😊
We will implement it.